docs/natspec-format.rst

```diff
@@ -82,10 +85,10 @@ Tag
 =========== =============================================================================== =============================
 ``@title``  A title that should describe the contract/interface                             contract, interface
 ``@author`` The name of the author                                                          contract, interface, function
-``@notice`` Explain to an end user what this does                                           contract, interface, function
-``@dev``    Explain to a developer any extra details                                        contract, interface, function
+``@notice`` Explain to an end user what this does                                           contract, interface, function, state variable
+``@dev``    Explain to a developer any extra details                                        contract, interface, function, state variable
 ``@param``  Documents a parameter just like in doxygen (must be followed by parameter name) function
-``@return`` Documents the return variables of a contract's function                         function
+``@return`` Documents the return variables of a contract's function                         function, state variable
```

Comment:
```suggestion
``@return`` Documents the return variables of a contract's function                         function, public state variable
```

Reply:
Thats true, `@return` should only be allowed on public state-variables.

Reply:
I added now a check: `@return` is now only allowed on public state-variables. I added also a test for this.